package.json

```diff
@@ -20,8 +20,13 @@
     "nodemon": "^1.18.7"
   },
   "dependencies": {
+    "@auth0/auth0-react": "^1.3.0",
     "bootstrap": "^4.6.0",
+<<<<<<< HEAD
```

Comment:
Hi Dustin,

There is a merge conflict here that probably you will need to resolve first. Can you take a look at it. I just want to make sure this will not break something